desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -89,23 +89,26 @@ def handle_on_message(channel_id, bot_id, text, user_id):
       if not response['ok']:
         raise PopupException(_("Error posting message"), detail=response["error"])
 
-
 def handle_on_link_shared(channel_id, message_ts, links):
   for item in links:
     path = urlsplit(item['url'])[2]
     id_type, qid_or_uuid = urlsplit(item['url'])[3].split('=')
```

Comment:
qid_or_uuid --> qid would just be more readable

Reply:
Fixed